frontend/packages/console-dynamic-plugin-sdk/src/api/api-types.ts

```diff
@@ -1,39 +0,0 @@
-import { K8sResourceCommon, K8sResourceKindReference, Selector } from '../extensions/console-types';
-
-export type WatchK8sResource = {
```

Comment:
I moved the content of this file to `extensions/console-types` because most of these types were duplicates.